package.json

```diff
@@ -43,25 +43,28 @@
   "license": "MIT",
   "dependencies": {
     "on-headers": "^1.0.1",
-    "pidusage": "^1.1.0",
-    "socket.io": "^1.5.1"
+    "pidusage": "^1.1.6",
+    "socket.io": "^2.0.3"
   },
   "scripts": {
     "coverage": "istanbul cover _mocha test --  --recursive",
-    "test": "mocha --recursive",
+    "test-ci": "mocha --recursive",
+    "test": "mocha --recursive --watch",
     "snyk-protect": "snyk protect",
+    "publish": "publish",
     "prepublish": "npm run snyk-protect",
-    "example": "cd examples && npm start",
+    "example": "cd examples && npm install && npm start",
```

Comment:
you need to do `npm i` first if you want to launch example otherwise you won't have `socket.io` available. also, I would recommend avoiding `cd` but use ` --prefix` instead:

```bash
npm i && npm i --prefix examples && npm start --prefix examples
```